src/domain/journey/subspace/pages/AdminSubspaceCommunityPage.tsx

```diff
@@ -83,6 +87,12 @@ const AdminSubspaceCommunityPage: FC<SettingsPageProps> = ({ routePrefix = '../'
             <CommunityApplications
               applications={applications}
               onApplicationStateChange={onApplicationStateChange}
+              canHandleInvitations
```

Comment:
I don't think you need this prop.

Reply:
It's required for the header of the block. Without it it says only Applications. When provided (value true) it states ' & invitations'